modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/EpisodeManagerImpl.kt

```diff
@@ -947,14 +947,12 @@ class EpisodeManagerImpl @Inject constructor(
     }
 
     override fun checkPodcastForEpisodeLimit(podcast: Podcast, playbackManager: PlaybackManager?) {
-        if (!podcast.overrideGlobalArchive) return
```

Comment:
This isn't needed because `podcast.autoArchiveEpisodeLimit` handles it on its own.